crates/sui-core/src/node_sync/node_state.rs

```diff
@@ -140,8 +140,9 @@ pub enum SyncArg {
     /// In follow mode, wait for 2f+1 votes for a tx before executing
     Follow(AuthorityName, ExecutionDigests),
 
-    /// Sync a cert which is appears as a parent in the verified effects of some other cert,
-    /// and is thus known to be final.
+    /// Sync a cert that is finalized, including its parents. It may
```

Comment:
This comment seems a bit misleading - the `including its parents` implies that other sync messages don't sync parents, but this is not true - all attempts to sync a cert will eventually sync the parents if necessary.

Reply:
fixed